Bottom: 6f7370a0ceda3ab1288b1c3ca2fc41896f3a8ced
Top:    57980b139203992502420839299e30f333266cb4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 13:56:09 -0500

Refresh of all-docker-image-repositories

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 2b99b556..3752cc00 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -58,9 +58,6 @@
     group: tape
     mode: 'u=rw,g=r,o='
 
-- name: copy script
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mantisbt/mantisbt.sh {{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}'
-
 - name: exists - state file
   stat:
     path: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/restore.date.txt'
@@ -76,6 +73,12 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-mantisbt'
 
+- name: copy script
+  copy:
+    remote_src: true
+    src: '{{ docker_projects_dir }}/docker-mantisbt/mantisbt.sh'
+    dest: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/'
+
 - name: build image
   docker_image:
     name: '{{ mantisbt_image_name }}'
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 7b65f235..d80e12ae 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -58,9 +58,6 @@
     group: tape
     mode: 'u=rw,g=r,o='
 
-- name: copy script
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mediawiki/mediawiki.sh {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}'
-
 - name: exists - state file
   stat:
     path: '{{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}/restore.date.txt'
@@ -77,6 +74,9 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-mediawiki'
 
+- name: copy script
+  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mediawiki/mediawiki.sh {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}'
+
 - name: build image
   docker_image:
     name: '{{ wiki_image_name }}'
